src/main/java/stsjorbsmod/powers/RepressedMemoryPower.java

```diff
@@ -0,0 +1,45 @@
+package stsjorbsmod.powers;
+
+import com.evacipated.cardcrawl.mod.stslib.StSLib;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.patches.ExertedField;
+import stsjorbsmod.patches.SelfExertField;
+
+public class RepressedMemoryPower extends CustomJorbsModPower {
+    public static final StaticPowerInfo STATIC = StaticPowerInfo.Load(RepressedMemoryPower.class);
+    public static final String POWER_ID = STATIC.ID;
+    //private final CardGroup originalPile;
+    private AbstractCard card;
+
+    public RepressedMemoryPower(AbstractPlayer owner,  AbstractCard card) {
+        super(STATIC);
+        this.owner = owner;
+        this.type = PowerType.DEBUFF;
```

Comment:
Preventable by artifacts and removable by Orange Pellets? Is that intended?

Reply:
Nope. Will change to buff.

Reply:
Actually, I think this should be the "SPECIAL" type of power we added, as we don't want a "buff" accidentally removed by a monster or a drawback from another card.